### PR TITLE
Adding Test-CsTeamsTranslationRule

### PR DESCRIPTION
**Pull request by JensTrier**
Adding Test-CsTeamsTranslationRule and updating New/Get/Set/Remove-CSTeamsTranslationRule.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsTeamsTranslationRule.md [hunk 1]
```diff
@@ -28,48 +34,58 @@ You can use this cmdlet to get an existing number manipulation rule (or list of
 
 ### Example 1
 ```powershell
-PS C:\> Get-CsTeamsTranslationRule
+Get-CsTeamsTranslationRule
 ```
 
-This command will show all translation rules that exist in the tenant. Identity, Description, Pattern and Translation parameters listed for each rule.
+This command will show all translation rules that exist in the tenant. Identity, Name, Description, Pattern and Translation parameters are listed for each rule.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsTranslationRule.md:40]:
```suggestion
This command will show all translation rules that exist in the tenant. Identity, Name, Description, Pattern, and Translation parameters are listed for each rule.

```

File: skype/skype-ps/skype/Get-CsTeamsTranslationRule.md [hunk 2]
```diff
@@ -28,48 +34,58 @@ You can use this cmdlet to get an existing number manipulation rule (or list of
...
 
 ### Example 2
 ```powershell
-PS C:\> Get-CsTeamsTranslationRule -Identity AddPlus1
+Get-CsTeamsTranslationRule -Identity AddPlus1
 ```
 
-This command will show Identity, Description, Pattern and Translation parameters for "AddPlus1" rule
+This command will show Identity, Name, Description, Pattern and Translation parameters for "AddPlus1" rule
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsTranslationRule.md:47]:
```suggestion
This command will show Identity, Name, Description, Pattern, and Translation parameters for the "AddPlus1" rule.

```

File: skype/skype-ps/skype/Get-CsTeamsTranslationRule.md [hunk 3]
```diff
@@ -28,48 +34,58 @@ You can use this cmdlet to get an existing number manipulation rule (or list of
...
+
+### Example 3
+```powershell
+Get-CsTeamsTranslationRule -Filter 'Add*'
+```
 
+This command will show Identity, Name, Description, Pattern and Translation parameters for all rules with Identity starting with Add.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsTranslationRule.md:54]:
```suggestion
This command will show Identity, Name, Description, Pattern, and Translation parameters for all rules with Identity starting with Add.

```

File: skype/skype-ps/skype/New-CsTeamsTranslationRule.md [hunk 4]
```diff
@@ -28,51 +34,51 @@ You can use this cmdlet to create a new number manipulation rule. The rule can b
...
 
 This example creates a rule that strips +1206555 from any E.164 ten digits number. For example, +12065555555 will be translated to 5555
 
 
 ## PARAMETERS
 
-### -Confirm
-Prompts you for confirmation before running the cmdlet.
+### -Identity
+The Identifier of the rule. This parameter is required and later used to assign the rule to Inbound or Outbound Trunk Normalization policy.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsTranslationRule.md:74]:
```suggestion
The Identifier of the rule. This parameter is required and later used to assign the rule to the Inbound or Outbound Trunk Normalization policy.

```

File: skype/skype-ps/skype/Set-CsTeamsTranslationRule.md [hunk 5]
```diff
@@ -17,36 +17,35 @@ Cmdlet to modify an existing normalization rule.
...
 This example modifies the rule that initially configured to strip +1206555 from any E.164 ten digits number. For example, +12065555555 translated to 5555 to a new pattern. Modified rule now only applies to three digit number (initially to four digits number) and adds one more number in prefix (+120655555 instead of +1206555)
 
-
-
 ## PARAMETERS
 
-### -Confirm
-Prompts you for confirmation before running the cmdlet.
+### -Identity
+Identifier of the rule. This parameter is required and later used to assign the rule to Inbound or Outbound Trunk Normalization policy.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsTranslationRule.md:40]:
```suggestion
Identifier of the rule. This parameter is required and later used to assign the rule to the Inbound or Outbound Trunk Normalization policy.

```

File: skype/skype-ps/skype/Test-CsTeamsTranslationRule.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+## SYNTAX
+
+### Test (Default)
+```powershell
+Test-CsTeamsTranslationRule [-PhoneNumber <string>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet tests a phone number against the configured number manipulation rules and return information about the matching rule.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Test-CsTeamsTranslationRule.md:25]:
```suggestion
This cmdlet tests a phone number against the configured number manipulation rules and returns information about the matching rule. 

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the minor suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the minor suggested changes. Many thanks.

Thanks & Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Pull request merged**